src/libkeymap/dump.c

```diff
@@ -315,18 +315,14 @@ void lk_dump_diacs(struct lk_ctx *ctx, FILE *fd)
 
 void lk_dump_keymaps(struct lk_ctx *ctx, FILE *fd)
 {
-	int i, n, m, s;
-	i = n = m = s = 0;
-
-	fprintf(fd, "keymaps");
+	int i, n, m, s, kw;
+	kw = i = n = m = s = 0;
 
 	for (i = 0; i < ctx->keymap->total; i++) {
-		if (ctx->keywords & LK_KEYWORD_ALTISMETA && i == (i | M_ALT))
```

Comment:
Why is it no longer needed?

Reply:
This code skipped (did not output) those keymaps that were ALT variants (i | M_ALT) if the ALTISMETA flag was set. That is, if alt_is_meta was active, ALT tables were not explicitly output, implying that they were automatically restored if alt_is_meta is specified. But dump_keymaps does not restore alt_is_meta itself.

Now the keymap dump always fully reflects all real tables, without the "magic" of silencing ALT tables. This makes the dump more transparent and unambiguous.